pbcommand/testkit/core.py

```diff
@@ -156,7 +156,7 @@ class PbTestScatterApp(PbTestApp):
     CHUNK_KEYS = ()
 
     def _to_rtc(self, tc, output_dir, tmp_dir):
-        return resolve_scatter_tool_contract(tc, self.INPUT_FILES, output_dir, tmp_dir, self.MAX_NPROC, self.TASK_OPTIONS, self.MAX_NCHUNKS, self.CHUNK_KEYS)
```

Comment:
Can we refrain from the kwarg usage here? Reasonably named vars make it clear what going on.